.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,111 @@
+name: "Bug Report"
```

Comment:
Wow, really impressed with how through this is! I like it but would also not be opposed to simplify + add fields/complexity over time. I'm good either way but just wanted to highlight in case others think there's a risk that this list of fields would be too overwhelming to users.

Reply:
Happy to simplify, but I have found that these fields request not only fairly basic information but very important information when it comes to understanding Kubernetes issues including Helm.

Reply:
Totally your call. We can always iterate if people are completing or we're just not getting the feedback we expected. That said, I've always found it a little easier to add more over time one you have real engagement.

Reply:
The reality is, in many cases we'll need this information anyway so I'd rather ask for it up front than chase people down. FYI, this bug template is almost a verbatim copy-and-paste of the one [I created originally for the Kyverno repository](https://github.com/kyverno/kyverno/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=bug-webhook.yaml&title=%5BBug%5D+), and in the couple hundred bugs logs since that was implemented I have never heard any feedback that it was requesting too much information. Let's go with this and dial it back if it somehow becomes an issue.